internal/services/apimanagement/migration/policy.go

```diff
@@ -20,11 +21,11 @@ type ApiManagementPolicyV0ToV1 struct{}
 
 func (ApiManagementPolicyV0ToV1) UpgradeFunc() pluginsdk.StateUpgraderFunc {
 	return func(ctx context.Context, rawState map[string]interface{}, meta interface{}) (map[string]interface{}, error) {
-		apiMgmtId, err := policy.ParseServiceID(rawState["id"].(string))
+		apiMgmtId, err := parse.ApiManagementID(rawState["id"].(string))
```

Comment:
I think these two IDs are identical and we don't need to revert this back to using the older generated parsers which imo we should be removing after doing sdk swaps.

Reply:
You are right! I'll change that up now. Thank you thank you